### DISCUSSION TITLE
publish from unity to spatial.io

### DISCUSSION THREAD

**Discussion by sarahforreals** [spatialsys/spatial-unity-sdk#20] [Help]
Hi there, I really need your help!
I just wanted to upload my Unity world, created and tested in unity and the try out sandbox in spatial. I have successfully logged in into my spatial.io account. so the connection is not the problem, but now when I click on publishing, as showed in the creator toolkit video, it says my package has successfully packed and it will take mostly 15 minutes to publish, maybe more, and that I will receive an e mail, if it was uploaded successfully. the thing is, I never received a notification, also after some hours, or days, I tried again 3 times, always with the same result, the support team did not answer me. So I wanted to ask you guys for your help!, do you know what I could do? or maybe know what could have went wrong?. There is also no notification inside of spatial
...

**Comment by AlanSpatial** [comment]
If you navigate to https://www.spatial.io/studio/packages you can check on the status of your packages. Have they failed maybe?
